README.md

```diff
@@ -1,24 +1,66 @@
-# README
+# My-Wallet
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This app is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-Things you may want to cover:
+## Getting Started
 
-* Ruby version
+To get a local copy of the project, follow these steps: 
+1. Open the terminal and navigate to the folder where you would like to put these files.
+2. Type the following commands into the terminal: 
+ ```
+ git clone https://github.com/Hadi111jafari/My-Wallet
+ ```
+ ```
+ cd My-Wallet
+ ```
 
-* System dependencies
+Check the ruby version being used and install it if it is not the same with that of the local machine
+Set up database direct them to the config/database.yml file
+```
+bundle install
+```
+```
+'npm install' or 'yarn install'
+```
+```
+rails db:create
+```
```

Comment:
- [ ] Kindly follow the documentation requirement and provide a video demo of your project, it should match these criterias:

![image](https://user-images.githubusercontent.com/65068771/190560265-55250963-2b37-4ef9-a403-0b778a9b4e70.png)